tests/test_coordinators.py

```
@@ -2134,33 +2134,92 @@ async def test_async_update_data(self, facade_coordinator: UnifiFacadeCoordinato
         self, facade_coordinator: UnifiFacadeCoordinator
     ):
-        """Test async request refresh."""
-        facade_coordinator._config_coordinator.async_request_refresh = AsyncMock()
-        facade_coordinator._device_coordinator.async_request_refresh = AsyncMock()
-        facade_coordinator._protect_coordinator.async_request_refresh = AsyncMock()
+        """
+        Test async_request_refresh forces a genuine refresh on each
+        sub-coordinator (async_refresh), not the debounced
+        async_request_refresh which can return before any fetch happens.
+        """
+        facade_coordinator._config_coordinator.async_refresh = AsyncMock()
```

Pylint (4.0.7)

[warning] Access to a protected member _config_coordinator of a client class

(W0212)